lib/search/index.ts

```diff
@@ -0,0 +1,106 @@
+import SearchWorker from 'worker-loader!./worker';
```

Comment:
Just curious, what is this syntax?

Reply:
That is a way to get a webpack loader working without having to configure webpack to handle the file type. In this case it runs the import through the webworker loader so that it turns it into something we can load in the worker frame